about_bootstrap.md

```diff
@@ -0,0 +1,29 @@
+## What you will learn in this section
+This section will basically help you to know what is bootstrap and why you should learn it.
+
+## About bootstrap
+Bootstrap, originally named Twitter Blueprint, was developed by Mark Otto and Jacob Thornton at Twitter as a framework to encourage consistency across internal tools. Here we will talk about why Bootstrap has capturted the developers attraction:
```

Comment:
remove - Here we will talk about why Bootstrap has capturted the developers attraction: